test/flipper_app/authorize_test.rb

```diff
@@ -0,0 +1,75 @@
+require_relative '../test_helper'
+require 'mocha/setup'
+require 'flipper_app/authorize'
+
+class AssignmentsApiTest < Minitest::Test
+  def test_unauthenticated_users_rejected
+    User.stubs(:where).returns(authorized_users)
```

Comment:
In this case this is harmless, but we should avoid stubbing `ActiveRecord` classes. We are returning an `Array` here, but `ActiveRecord` returns an `ActiveRecord_Relation`.